deploy/quickstart-k8s.yaml

```
@@ -1,9 +1,54 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: everest-admin
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: everest-admin-role
+rules:
+  - apiGroups: ["everest.percona.com"]
+    resources: ["*"]
+    verbs: ["*"]
+  - apiGroups: ["operator.victoriametrics.com"]
+    resources: ["vmagents"]
+    verbs: ["*"]
+  - apiGroups: [""]
+    resources: ["namespaces", "secrets"]
+    verbs: ["*"]
+  - apiGroups: [""]
+    resources: ["nodes", "pods", "persistentvolumes"]
+    verbs: ["get", "list"]
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["storageclasses"]
+    verbs: ["list"]
```

These need to be cluster role rules
```suggestion
---
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
metadata:
  name: everest-admin-cluster-role
rules:
  - apiGroups: [""]
    resources: ["nodes", "pods", "persistentvolumes"]
    verbs: ["get", "list"]
  - apiGroups: ["storage.k8s.io"]
    resources: ["storageclasses"]
    verbs: ["list"]
---
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  name: everest-admin-cluster-role-binding
roleRef:
  kind: "ClusterRole"
  apiGroup: "rbac.authorization.k8s.io"
  name: everest-admin-cluster-role
subjects:
  - kind: "ServiceAccount"
    name: everest-admin
```